### PR TITLE
Add a changelog for the 1.21.0 release

### PR DESCRIPTION
**Pull request by vigh-m**
**Description of changes:**
Update the changelog for the 1.21.0 release

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,90 @@
...
+* Update bottlerocket-core-kit to v2.3.1 ([#4122])
+* Migrate to core kit ([#4060])
+* Update bottlerocket-sdk to 0.43 ([#4122])
+
+### Settings Extensions
+
+* Use settings models vended by bottlerocket-settings-sdk ([#4057])
+* Migrate to settings plugins and eliminate variant-based conditional compilation ([#4038])
+* Enable settings extensions for bootstrap containers, network, host containers, autoscaling, container runtime, and OCI defaults ([#4050])
+* update to bottlerocket-settings-models v0.2.0 ([#4118])
```
Comment (koooosh) [inline CHANGELOG.md]:
```suggestion
* Update to bottlerocket-settings-models v0.2.0 ([#4118])
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,90 @@
...
+* Update host containers ([#4117])
+
+## Orchestrator Changes
+
+### Kubernetes
+
+* Enable k8s reserved cpus ([#3964])
+* Drop k8s 1.27 metal and VMware variants ([#4079])
+* Drop k8s 1.26 metal and VMware variants ([#4018]) 
+* Build the pause image from upstream and use as the default ([#3940]) thanks @tzneal!
```
Comment (koooosh) [inline CHANGELOG.md]: nit: Seems too verbose, how about just:
```
Build pause image from upstream
```

Comment (koooosh) [reply CHANGELOG.md]: Also, let's keep the acknowledgements formatting consistent:

```
... ([#3940]) - Thanks @tzneal!
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,90 @@
...
+
+* Archive previous release migrations ([#4014])
+* Update Go dependencies ([#3999])
+
+### Twoliter
+
+* Remove leftover vendor section ([#4071])
+* Update Twoliter to 0.4.4 ([#4008], [#4086], [#4093], [#4123]) 
+* Update bottlerocket-core-kit to v2.3.1 ([#4122])
+* Migrate to core kit ([#4060])
```
Comment (koooosh) [inline CHANGELOG.md]: nit: Not sure if ordering matters, but in my head, it makes sense to reference the core-kit migration first (#4060) before mentioning the updates to it

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,90 @@
...
+* Update bottlerocket-sdk to 0.43 ([#4122])
+
+### Settings Extensions
+
+* Use settings models vended by bottlerocket-settings-sdk ([#4057])
+* Migrate to settings plugins and eliminate variant-based conditional compilation ([#4038])
+* Enable settings extensions for bootstrap containers, network, host containers, autoscaling, container runtime, and OCI defaults ([#4050])
```
Comment (koooosh) [inline CHANGELOG.md]: nit: Not familiar with this change, but can it just be shortened to:

```
Enable settings extensions
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,90 @@
...
+## Platform Changes
+
+### AWS
+
+* Add udev rule to create symlinks using EBS volumes’ device names ([#3977])
+
+## Package changes
+
+* Add Neuron kmod for 6.1 kernel ([#3982])
+* Update containerd to 1.7.20 via bottlerocket-core-kit ([#4122])
```
Comment (koooosh) [inline CHANGELOG.md]: nit: Can this just be:


```
Update containerd to 1.7.20 
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,90 @@
...
+* Fix OpenAPI spec link ([#4062])
+* Fix NVIDIA variants in SSM parameters ([#4047])
+* Add k8s command to retrieve log archive ([#3993])
+* Fix netdog reference link ([#3974]) - Thanks @emmanuel-ferdman!
+* Update BUILDING.md with the latest Docker requirements ([#4098])
+
+[#3940]: https://github.com/bottlerocket-os/bottlerocket/pull/3940
+[#3964]: https://github.com/bottlerocket-os/bottlerocket/pull/3964
+[#3977]: https://github.com/bottlerocket-os/bottlerocket/pull/3977
+[#3974]: https://github.com/bottlerocket-os/bottlerocket/pull/3974
```
Comment (koooosh) [inline CHANGELOG.md:62]: nit: This should come before #3977

**Review by koooosh — COMMENTED** [review]
Left comments

**Comment by vigh-m** [comment]
Update addressing comments

**Review by larvacea — APPROVED** [review]
Looks good to me. I looked at each pull request in this README just to be sure that they are in fact pull requests, and will be present in this release.

**Review by koooosh — APPROVED** [review]
LGTM, thanks for the fixes

**Review by ytsssun — APPROVED** [review]
LGTM

**Pull request merged**